site/data/dynamo.ts

```diff
@@ -0,0 +1,30 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, QueryCommand } from "@aws-sdk/lib-dynamodb";
+import { ptSituationElementSchema } from "@create-disruptions-data/shared-ts/siriTypes.zod";
+import logger from "../utils/logger";
+
+export const getDdbClient = (): DynamoDBClient => new DynamoDBClient({ region: "eu-west-2" });
+
+export const getDdbDocumentClient = (ddbClient = getDdbClient()): DynamoDBDocumentClient =>
+    DynamoDBDocumentClient.from(ddbClient);
+
+export const getDisruptionsDataFromDynamo = async (ddbDocClient: DynamoDBDocumentClient, tableName: string) => {
```

Comment:
We could also just define tableName in this file since it'll always be the same table